src/User/User.php

```diff
@@ -111,6 +114,51 @@ public function setIdentity(IdentityInterface $identity): void
         $this->identity = $identity;
     }
 
+    /**
+     * Return the auth key value
+     *
+     * @return string Auth key value
+     */
+    public function getAuthKey(): string
+    {
+        return 'ABCD1234';
+    }
+
+    /**
+     * Validate auth key
+     *
+     * @param String $authKey Auth key to validate
+     * @return bool True if is valid
+     */
+    public function validateAuthKey($authKey): bool
+    {
+        return $authKey === 'ABCD1234';
+    }
+
+    /**
+     * Sends an identity cookie.
+     *
+     * @param IdentityInterface $identity
+     * @param int $duration number of seconds that the user can remain in logged-in status.
+     */
+    protected function sendIdentityCookie(IdentityInterface $identity, int $duration): void
+    {
+        $data = json_encode(
+            [
+                $identity->getId(),
+                $this->getAuthKey(),
+                $duration,
+            ],
+            JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE
+        );
+
+        $expireDateTime = new \DateTimeImmutable();
+        $expireDateTime->setTimestamp(time() + $duration);
+        $cookieIdentity = (new Cookie('remember', $data))->expireAt($expireDateTime);
```

Comment:
It is a good idea to make cookie name configurable.

Reply:
I updated my PR with a new property for make this configurable.